packages/core/src/components/popover/_common.scss

```diff
@@ -161,6 +161,11 @@ $dark-popover-background-color: $dark-gray3 !default;
 
   .#{$ns}-popover-arrow-fill {
     fill: $background-color;
+
+    @media (forced-colors: active) and (prefers-color-scheme: dark) {
+      // Windows High Contrast dark theme
+      fill: $pt-high-contrast-mode-border-color;
```

Comment:
minor: we may want to apply the same style to tooltips with intents, otherwise we get this:

<img width="198" alt="image" src="https://user-images.githubusercontent.com/723999/192384053-987da2fa-5c65-4101-9ad5-3cf5c259cadd.png">

Reply:
whoops good call